doc/pcre2.3

```diff
@@ -186,17 +186,20 @@ In the "man" and HTML formats, there is also a short page for each C library
 function, listing its arguments and results.
 .
 .
-.SH AUTHOR
+.SH AUTHORS
 .rs
 .sp
-.nf
-Philip Hazel
-Retired from University Computing Service
-Cambridge, England.
-.fi
+The current maintainers of PCRE2 are Nicholas Wilson and Zoltan Herczeg.
```

Comment:
I would have used Zoltán's actual name (with á) but that causes difficulties inside manpages. It's totally possible on Linux, by using the `\[u00E1]` escape, but I don't know about how that would be handled on other systems. So, I'm keeping the manpages ASCII-only, out of caution.